src/services/message/index.ts

```diff
@@ -0,0 +1,89 @@
+import { messageConfig, socketConfig } from '../../config/socket';
+import { io } from '../../index';
+import { Message } from 'redis-smq';
+
+export const triggerMessageHandler = async (message: Message, cb: (err?: Error) => void) => {
+  const body = message.getBody();
+  if (!body) throw new Error('body not found');
+  const messageId = message.getId();
+  if (!messageId) throw new Error('message not found');
+
+  messageConfig[messageId] = {
```

Comment:
create a function to set the status of message to something, so that if in future states are persisted in db , it can be changed at single place.